src/staketaxcsv/atom/api_lcd.py

```diff
@@ -46,7 +46,7 @@ def account_exists(wallet_address):
 def _get_txs(wallet_address, is_sender, offset, sleep_seconds):
     uri_path = "/cosmos/tx/v1beta1/txs"
     query_params = {
-        "order_by": "ORDER_BY_DESC",
```

Comment:
Unsure if this is necessary right now? But maybe if the cosmos hub goes to v0.46.x it will be?